addons/account/models/reconciliation_widget.py

```diff
@@ -286,22 +290,23 @@ def get_data_for_manual_reconciliation(self, res_type, res_ids=None, account_typ
                         {4}
                         {5}
                         AND l.company_id = {6}
-                        AND EXISTS (
+                        AND ({7} OR (
```

Comment:
add an in-line comment to depict the use case and what this aml_ids argument does